ui/Screen/Lock.svelte

```diff
@@ -28,20 +28,28 @@
     display: flex;
     flex-direction: column;
   }
+  .form {
+    display: flex;
+    margin-top: 1.5rem;
+  }
 </style>
 
 <div class="lock">
-  <Input.Text
-    autofocus
-    placeholder="Enter your passphrase"
-    bind:value={passphrase}
-    dataCy="passphrase-input"
-    on:keydown={onKeydown}
-    style="width: 16rem; margin-bottom: 2rem;" />
-  <Button
-    dataCy="unlock-button"
-    disabled={passphrase.length === 0}
-    on:click={unlock}>
-    Unlock
-  </Button>
+  <Emoji emoji={'🚪'} size="huge" style="margin-bottom: 0.75rem;" />
+  <h3>Unlock the app</h3>
+  <div class="form">
+    <Input.Password
+      autofocus
+      placeholder="Enter your passphrase"
+      bind:value={passphrase}
+      dataCy="passphrase-input"
+      on:keydown={onKeydown}
```

Comment:
`Input.Password` already does the keydown-enter event handling, which we could use here.

So you could do here: `on:enter={onEnter}` and then the function in this file could be reduced to:
`const onEnter = () => {
    if (passphrase.length > 0) {
      unlock();
    }
  };`